mkdocs.yml

```diff
@@ -20,12 +20,12 @@ nav:
       - Azure: hopsworksai/azure/restrictive_permissions.md
     - Cluster creation:
       - AWS: hopsworksai/aws/cluster_creation.md
+      - AWS (EKS and ECR): hopsworksai/aws/eks_ecr_integration.md
```

Comment:
Can you rename to `Integration with EKS and ECR` right now I find it confusing to have two AWS

Reply:
Fixes